datahub-web-react/src/app/shared/deleteUtils.ts

```diff
@@ -23,6 +26,12 @@ export const getEntityProfileDeleteRedirectPath = (type: EntityType) => {
         case EntityType.GlossaryTerm:
             // Return to glossary page.
             return '/glossary';
+        case EntityType.DataProduct:
```

Comment:
Nice so data products are ALWAYS part of a domain

Reply:
yup exactly